docs/transport/s3.md

```diff
@@ -0,0 +1,28 @@
+# transport: s3
+
+The S3 transporter puts logical WAL messages as *gzipped* objects in an S3 bucket partitioned by year, month and day in increasing order of time.
```

Comment:
Call this out explicitly that the time in the path is time at which the data is read and not when it was created.